databricks_cli/sdk/api_client.py

```diff
@@ -70,8 +72,16 @@ def __init__(self, user=None, password=None, host=None, token=None,
         if host[-1] == "/":
             host = host[:-1]
 
+        retries = Retry(
+            total=6,
+            backoff_factor=1,
+            status_forcelist=[429],
+            method_whitelist=set({'POST'}) | set(Retry.DEFAULT_METHOD_WHITELIST),
```

Comment:
Please use `DEFAULT_ALLOWED_METHODS`, because `DEFAULT_METHOD_WHITELIST` is deprecated.

Reply:
The new constant is not available in urllib3 `v1.25.10`, which seems to be installed with databricks-cli by default. It was introduced in `v1.26.0`. Confirmed by trying to use it:
```
Error: AttributeError: type object 'Retry' has no attribute 'DEFAULT_ALLOWED_METHODS'
```

Reply:
ok, then we need to schedule version updates, as this code would have to be updated down the line anyway